test/Driver/linker-args-order-linux.swift

```diff
@@ -0,0 +1,11 @@
+// Statically link a "hello world" program
+// REQUIRES: OS=linux-gnu
+// REQUIRES: static_stdlib
+print("hello world!")
+// RUN: %empty-directory(%t)
+// RUN: %target-swiftc_driver -v -static-stdlib -o %t/static-stdlib %s -Xlinker --no-allow-multiple-definition 2>&1| %FileCheck %s
```

Comment:
You shouldn't need to actually run the linker here. Instead, you can use `-###` (or `-driver-print-jobs`) to show what would be run. That way we can run the test on other platforms.